doc/api/fs.md

```diff
@@ -3918,6 +3918,33 @@ On Linux, positional writes do not work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
+#### filehandle.write(string[, position[, encoding]])
+<!-- YAML
+added: v10.0.0
+-->
+
+* `string` {string}
+* `position` {integer}
+* `encoding` {string} **Default:** `'utf8'`
+* Returns: {Promise}
+
+Write `string` to the file. If `string` is not a string, then
+the value will be coerced to one.
+
+`position` refers to the offset from the beginning of the file where this data
+should be written. If `typeof position !== 'number'` the data will be written at
```

Comment:
Could this be simply written as, "If the type of position is not a number"?

Reply:
So I basically copied the same wording used here: https://github.com/nodejs/node/pull/23224/files#diff-acabf706a8aa070a8796e3573f7e4678R3910

This style is used in the callback APIs too. I figured I would use the same wording that's used in those other places. I'd be happy to clarify those other places also if you think that's a good idea.